
From: Jens Axboe <axboe@suse.de>

This easily clashes with others (it's been seen, hence the patch), so lets
move aacraid log levels to something a little more private.

Signed-off-by: Jens Axboe <axboe@suse.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/scsi/aacraid/aacraid.h |   18 +++++++++---------
 25-akpm/drivers/scsi/aacraid/commsup.c |    2 +-
 2 files changed, 10 insertions(+), 10 deletions(-)

diff -puN drivers/scsi/aacraid/aacraid.h~reduce-aacraid-namespace-pollution drivers/scsi/aacraid/aacraid.h
--- 25/drivers/scsi/aacraid/aacraid.h~reduce-aacraid-namespace-pollution	Mon Aug  9 14:13:17 2004
+++ 25-akpm/drivers/scsi/aacraid/aacraid.h	Mon Aug  9 14:13:17 2004
@@ -404,15 +404,15 @@ struct aac_init
 };
 
 enum aac_log_level {
-	LOG_INIT			= 10,
-	LOG_INFORMATIONAL		= 20,
-	LOG_WARNING			= 30,
-	LOG_LOW_ERROR			= 40,
-	LOG_MEDIUM_ERROR		= 50,
-	LOG_HIGH_ERROR			= 60,
-	LOG_PANIC			= 70,
-	LOG_DEBUG			= 80,
-	LOG_WINDBG_PRINT		= 90
+	LOG_AAC_INIT			= 10,
+	LOG_AAC_INFORMATIONAL		= 20,
+	LOG_AAC_WARNING			= 30,
+	LOG_AAC_LOW_ERROR		= 40,
+	LOG_AAC_MEDIUM_ERROR		= 50,
+	LOG_AAC_HIGH_ERROR		= 60,
+	LOG_AAC_PANIC			= 70,
+	LOG_AAC_DEBUG			= 80,
+	LOG_AAC_WINDBG_PRINT		= 90
 };
 
 #define FSAFS_NTC_GET_ADAPTER_FIB_CONTEXT	0x030b
diff -puN drivers/scsi/aacraid/commsup.c~reduce-aacraid-namespace-pollution drivers/scsi/aacraid/commsup.c
--- 25/drivers/scsi/aacraid/commsup.c~reduce-aacraid-namespace-pollution	Mon Aug  9 14:13:17 2004
+++ 25-akpm/drivers/scsi/aacraid/commsup.c	Mon Aug  9 14:13:17 2004
@@ -761,7 +761,7 @@ void aac_printf(struct aac_dev *dev, u32
 		length = 255;
 	if (cp[length] != 0)
 		cp[length] = 0;
-	if (level == LOG_HIGH_ERROR)
+	if (level == LOG_AAC_HIGH_ERROR)
 		printk(KERN_WARNING "aacraid:%s", cp);
 	else
 		printk(KERN_INFO "aacraid:%s", cp);
_
